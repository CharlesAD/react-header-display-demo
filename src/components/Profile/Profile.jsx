import PropTypes from "prop-types";

export default function Profile({ name, imgUrl }) {
  return (
    <img
      src={imgUrl}
      alt={name}
      className="mx-auto w-24 items-center rounded-full pb-4"
    />
  );
}

Profile.propTypes = {
  name: PropTypes.string.isRequired,
  imgUrl: PropTypes.string.isRequired,
};
