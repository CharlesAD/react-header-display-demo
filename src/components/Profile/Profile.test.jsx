import renderer from "react-test-renderer";
import Profile from "./Profile";

it("renders without crashing", () => {
  const rendered = renderer.create(<Profile />).toJSON();
  expect(rendered).toMatchSnapshot();
});
