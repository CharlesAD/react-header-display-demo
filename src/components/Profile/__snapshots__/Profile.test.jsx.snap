// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<img
  className="mx-auto w-24 items-center rounded-full pb-4"
/>
`;
